src/Plugin/Filter/MediaFileLinkFilter.php

```diff
@@ -43,14 +43,24 @@ public function process($text, $langcode): FilterProcessResult
         $xpath = new \DOMXPath($dom);
         $storage = $this->entityTypeManager->getStorage('media');
 
-        foreach ($xpath->query('//a[@data-media-file-link]') as $element) {
```

Comment:
@akasake I think it's possible to be fully backwards compatible with our ckeditor4 plugin \_and\_ `ckeditor5_entity_browser`.

Still (also) search for `data-media-file-link` and extract the `$mid` from there like we used to.